### DISCUSSION TITLE
Runecrafting Utilities Addition

### DISCUSSION THREAD

**Discussion by Senken12** [runelite/runelite#18087] [Ideas]
It would be nice if max cape's left click could be changed to crafting guild when next to a runecrafting altar (similar to how the essence pouches change to empty when next to one)
Could also be extended to include diary cape swapping to Jarr (Desert)

**Comment by YvesW** [comment]
We're not the authors of Runecrafting Utilities.

Support for hub plugins should be directed to the author of the plugin. You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
